tools/cli/go-whisk/whisk/wskerror.go

```diff
@@ -34,6 +34,7 @@ const NO_DISPLAY_PREFIX         bool = false
 const APPLICATION_ERR           bool = true
 const NO_APPLICATION_ERR        bool = false
 const TIMED_OUT                 bool = true
+const UNABLE_CREATE_ACTION_ERR  string = "Unable to create action '{{.name}}': {{.err}}"
```

Comment:
Ok, thank you for your help @markusthoemmes , seems I don't t get the real spirit of of the issue yet.